themes/mongodb/layout.html

```diff
@@ -128,7 +128,7 @@
     <title>{{ title|striptags|e }}</title>
   {%- endblock -%}
 
-  <link rel="shortcut icon" href="https://docs.mongodb.com/assets/favicon.ico" />
+  <link rel="shortcut icon" href="{{theme_base_url}}/assets/favicon.ico" />
```

Comment:
The favicon may be absent on the staging links, but should be available once mongodb.com/docs is launched. This asset is already present in the dotcom prod bucket